modules/Users/entryPointSaveGoogleApiKey.php

```diff
@@ -0,0 +1,58 @@
+<?php
+if(!defined('sugarEntry') || !sugarEntry) die('Not A Valid Entry Point');
```

Comment:
PSR-2 issue, Can you please use braces '{ }' for if statements.

Reply:
Fixing